packages/preview/fireside/1.0.0/template/main.typ

```diff
@@ -0,0 +1,30 @@
+#import "@preview/fireside:1.0.0": fireside
+
+#show: fireside.with(
+  title: [Anakin \ Skywalker],
+  from_details: [
```

Comment:
I recommend naming things in kebab-case, i.e. `from-details`, as it's the idiomatic casing convention for Typst.

Reply:
Good catch!